src/zen/forks/fork.cpp

```
@@ -59,14 +59,18 @@ void Fork::setCommunityFundAddressMap(const std::map<CBaseChainParams::Network,s
         printf("Fork attempting to set communityFundAddress map of the wrong size! communityFundAddressMap.size()=%ld MAX_NETWORK_TYPES=%d\n",communityFundAddressMap.size(),CBaseChainParams::Network::MAX_NETWORK_TYPES);
         assert(0);
     }
-    if (cfType == CommunityFundType::FOUNDATION) {
+    switch (cfType) {
+    case CommunityFundType::FOUNDATION:
         this->communityFundAddressMap = communityFundAddressMap;
-    }
-    if (cfType == CommunityFundType::SECURENODE) {
+        break;
+    case CommunityFundType::SECURENODE:
         this->secureNodeFundAddressMap = communityFundAddressMap;
-    }
-    if (cfType == CommunityFundType::SUPERNODE) {
+        break;
+    case CommunityFundType::SUPERNODE:
         this->superNodeFundAddressMap = communityFundAddressMap;
+        break;
+    default:
+        break;
```

this should throw an exception.